xinference/api/restful_api.py

```diff
@@ -572,7 +617,7 @@ async def create_embedding(self, request: CreateEmbeddingRequest):
 
         try:
             embedding = await model.create_embedding(request.input)
-            return embedding
+            return Response(embedding, media_type="application/json")
```

Comment:
Is this `media_type="application/json"` neccessary? All other interfaces don't specify this option, should it be the default value?

Reply:
It is required because this is a `Respone` not a `JSONResponse`. The embedding responses are serialized in the ModelActor to offload the pressure of fast api.